### PR TITLE
Add 'WIP' filter

### PR DESCRIPTION
**Pull request by GuiRodriguero**
Development of issue #400 
This is my first Pull Request on GitHub, I hope it helps! I'm open to any discuss/comments about this feature

### CODE REVIEW

File: src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/filter/ShowWIPFilter.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    @Nullable
+    @Override
+    public String getSearchQueryPart() {
+        return value ? null : "-is:wip";
+    }
+
+    public final class ShowWIPActionFilter extends ToggleAction implements DumbAware {
+        public ShowWIPActionFilter() {
+            super("WIP changes", "WIP changes", Git4ideaIcons.DeepCompare);
```
Comment (GuiRodriguero) [inline src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/filter/ShowWIPFilter.java]: I didn't find any other icon in the project that would be a better fit for this filter... toughts?

**Comment by GuiRodriguero** [comment]
This is my first Pull Request on GitHub, I hope it helps! I'm open to any discuss/comments about this feature

File: src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/filter/ShowWIPFilter.java (hunk 1 above)
Comment (uwolfer) [inline src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/filter/ShowWIPFilter.java]: For the first argument `text` IntelliJ shows the following warning:
> String 'WIP changes' is not properly capitalized. It should have title capitalization 

For the second argument `description`, you could probably use "Display WIP changes".

Comment (uwolfer) [reply src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/filter/ShowWIPFilter.java]: What do you think about using the icon `AllIcons.Actions.Resume` (I have not checked it that one is available in the 2026 branch yet)?

Nice tool: https://jetbrains.design/intellij/resources/icons_list/

**Review by vecnajeffli — APPROVED** [review]

**Comment by uwolfer** [comment]
@GuiRodriguero Do you plan to integrate my review feedback?

**Comment by GuiRodriguero** [comment]
> @GuiRodriguero Do you plan to integrate my review feedback?

@uwolfer Yes, I just made the new commit with your suggestions. I was not aware of this icon list (thanks for sharing). After looking up some icons, I decided to use the following for the filter. What do you think?

image
image

**Pull request merged**